src/rpc/protocol/query.rs

```diff
@@ -532,6 +544,19 @@ impl ProtoEncoder for ObTableQuery {
             buf.put_slice(HTABLE_FILTER_DUMMY_BYTES);
         }
 
+        util::encode_vi64(self.scan_range_columns.len() as i64, buf)?;
+        for s in &self.scan_range_columns {
+            util::encode_vstring(s, buf)?;
+        }
+
+        util::encode_vi64(self.aggregations.len() as i64, buf)?;
+        for agg in &self.aggregations {
+            util::encode_vi64(1, buf)?;
```

Comment:
Why encode 1?

Reply:
we should encode 1 for the version

Reply:
I guess we could use:
```
    fn encode_header(&self, buf: &mut BytesMut) -> Result<()> {
        let len = self.len()?;
        buf.reserve(len);
        util::encode_vi64(VERSION, buf)?;
        util::encode_vi64(self.content_len()? as i64, buf)?;

        Ok(())
    }
```
Pls check whether we could use this func from ``trait ObPayload`` or not.

Reply:
fixed, use payload